include/xcshell/build_in/cd.h

```diff
@@ -5,13 +5,15 @@
 #include <vector>
 
 #include "./build_in_command.h"
+#include "xcshell/error_handling.h"
 
 class Cd : public BuildInCommand {
  public:
   int Execute(const std::vector<std::string> &args, std::ostream &os,
               std::ostream &os_err);
   std::string GetName() { return "cd"; }
   ~Cd() {}
+  ErrorHandling error_handling_;
```

Comment:
Do we need instance for `ErrorHandling`? Since all the public member is static, which means we can call it directly by `ErrorHandling:: ErrorSelector`.